StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/GenericInfoScreen/GenericInfoBodyView.swift

```diff
@@ -26,9 +26,8 @@ func GenericInfoBodyView(
                 textBodyEntry,
                 didSelectURL: didSelectURL
             )
-        case .image(let image):
-            _ = image
-            entryView = nil // TODO(kgaidis): implement ImageBodyEntry support
+        case .image(let imageBodyEntry):
+            entryView = ImageBodyEntryView(imageBodyEntry)
```

Comment:
note that I don't think this is yet used in prod...so most of this implementation is a bit of guess work of what's a good-enough V1